In [2]:
#Modified from Gensim Doc2Vec Tutorial 

import sys
sys.path.append("/Users/noah/Flatiron/FinalProject/CaseLawCorpusMatcher/src/gensim/")
from alt_models import dbow
from model_workflow import *

model = dbow()

data = prep_model(model,"/Users/noah/Flatiron/FinalProject/data/Lines/aklines.txt")

In [3]:
train(model,data,100)

In [6]:
evaluate(model,data)

Counter({0: 199, 1: 1})                


In [5]:
model.save("present")

/Users/noah/anaconda3/envs/learn-env/lib/python3.6/site-packages/smart_open/smart_open_lib.py:398: UserWarning: This function is deprecated, use smart_open.open instead. See the migration notes for details: https://github.com/RaRe-Technologies/smart_open/blob/master/README.rst#migrating-to-the-new-open-function
  'See the migration notes for details: %s' % _MIGRATION_NOTES_URL


In [16]:
# Pick a random document from the corpus and infer a vector from the model
doc_id = random.randint(0, len(data) - 1)
inferred_vector = model.infer_vector(data[doc_id].words)
sims = model.docvecs.most_similar([inferred_vector], topn=len(model.docvecs))

# Compare and print the most/median/least similar documents from the train corpus
print('Test Document ({}): «{}»\n'.format(doc_id, ' '.join(data[doc_id].words)))
print(u'SIMILAR/DISSIMILAR DOCS PER MODEL %s:\n' % model)
for label, index in [('MOST', 0),('SECOND MOST', 1), ('MEDIAN', len(sims)//2), ('LEAST', len(sims) - 1)]:
    print(u'%s %s: «%s»\n' % (label, sims[index], ' '.join(data[sims[index][0]].words)))

Test Document (51063): «appellees owners of acres of land with dwelling thereon adjoining appellantâ right of way brought this suit to recover damages in the amount of for the closing or obstructing of road crossing over appellantâ railroad and right of way leading to ap pellees property in the town of buckner arkansas appellant denied any liability on jury trial and at the close of all the testimony the court on its own motion and over appellantâ objections and exceptions instructed the jury as matter of law that from the evidence adduced appellees had in effect by long continued adverse use of the crossing in question acquired prescriptive right in said crossing the instruction no contained this recital gentlemen of the jury this suit is brought by the plaintiffs against the defendant railroad company for what the plaintiffs allege was the wrongful act of the railroad company in making unfit for use certain crossing that had been used by the plaintiff and his tenants and others desir

Test Document (51063): «appellees owners of acres of land with dwelling thereon adjoining appellantâ right of way brought this suit to recover damages in the amount of for the closing or obstructing of road crossing over appellantâ railroad and right of way leading to ap pellees property in the town of buckner arkansas appellant denied any liability on jury trial and at the close of all the testimony the court on its own motion and over appellantâ objections and exceptions instructed the jury as matter of law that from the evidence adduced appellees had in effect by long continued adverse use of the crossing in question acquired prescriptive right in said crossing the instruction no contained this recital gentlemen of the jury this suit is brought by the plaintiffs against the defendant railroad company for what the plaintiffs allege was the wrongful act of the railroad company in making unfit for use certain crossing that had been used by the plaintiff and his tenants and others desiring to use it for long period of years there isnâ any evidence that it was public crossing as the term is commonly used or that it was private crossing as the term is used by the railroad company but all the evidence is to the effect that this place on the railroad right of way was usable by the plaintiff and by his tenants and by others desiring to go across the railroad at that point for long eriod of years and that beginning in or thereabouts the railroad company for more than twenty years has maintained that spot as crossing that presents an issue to the court as to whether or not this suit could be maintained by the plaintiffs the defendant railroad company has defended the suit so far as that feature is concerned on the ground that such use was with its permission and it therefore could withdraw that permission at any time it saw fit the plaintiffs contend in view of the fact there was no other means of ingress and egress to such premises the plaintiffsâ sole and only method of going to and from that place is across this place the court is inclined to the view that the plaintiffs are entitled to recover in this case following this instruction the court proceeded to submit the question of the amount of damages only for the juryâ determination the court erred in giving instruction no above it appears undisputed that the town of buckner had never dedicated any street over this crossing appellees use was not based upon any consideration they did not claim any statutory right to its use under ark stats or we do not detail the testimony it suffices to say that there was evidence on the part of appellant tending to show that the use of this crossing by appellees and the public was permissive only and on the other hand evidence on the part of appellees of its adverse use for more than seven years or that they had used it as matter of legal right and not as matter of permission this was jury question the rule is well established that permissive use cannot ripen into legal right merely by lapse of time mcgill miller ark we said in britt berry ark the rule is that where the entry is permissive the statute will not begin to run against the legal owner until an adverse holding is declared and notice of such change is brought to the knowledge of the owner on the other hand it is also well established that the statute of limitations operates against railroad corporations whose lands are held adversely as well as against individuals and this applies to the right of way st louis san francisco railroad company ruttan ark for the error indicated the judgment is reversed and the cause remanded for new trial»

SIMILAR/DISSIMILAR DOCS PER MODEL Doc2Vec(dbow,d300,n5,mc10,s0.001,t4):

MOST (51063, 0.5057280659675598): «appellees owners of acres of land with dwelling thereon adjoining appellantâ right of way brought this suit to recover damages in the amount of for the closing or obstructing of road crossing over appellantâ railroad and right of way leading to ap pellees property in the town of buckner arkansas appellant denied any liability on jury trial and at the close of all the testimony the court on its own motion and over appellantâ objections and exceptions instructed the jury as matter of law that from the evidence adduced appellees had in effect by long continued adverse use of the crossing in question acquired prescriptive right in said crossing the instruction no contained this recital gentlemen of the jury this suit is brought by the plaintiffs against the defendant railroad company for what the plaintiffs allege was the wrongful act of the railroad company in making unfit for use certain crossing that had been used by the plaintiff and his tenants and others desiring to use it for long period of years there isnâ any evidence that it was public crossing as the term is commonly used or that it was private crossing as the term is used by the railroad company but all the evidence is to the effect that this place on the railroad right of way was usable by the plaintiff and by his tenants and by others desiring to go across the railroad at that point for long eriod of years and that beginning in or thereabouts the railroad company for more than twenty years has maintained that spot as crossing that presents an issue to the court as to whether or not this suit could be maintained by the plaintiffs the defendant railroad company has defended the suit so far as that feature is concerned on the ground that such use was with its permission and it therefore could withdraw that permission at any time it saw fit the plaintiffs contend in view of the fact there was no other means of ingress and egress to such premises the plaintiffsâ sole and only method of going to and from that place is across this place the court is inclined to the view that the plaintiffs are entitled to recover in this case following this instruction the court proceeded to submit the question of the amount of damages only for the juryâ determination the court erred in giving instruction no above it appears undisputed that the town of buckner had never dedicated any street over this crossing appellees use was not based upon any consideration they did not claim any statutory right to its use under ark stats or we do not detail the testimony it suffices to say that there was evidence on the part of appellant tending to show that the use of this crossing by appellees and the public was permissive only and on the other hand evidence on the part of appellees of its adverse use for more than seven years or that they had used it as matter of legal right and not as matter of permission this was jury question the rule is well established that permissive use cannot ripen into legal right merely by lapse of time mcgill miller ark we said in britt berry ark the rule is that where the entry is permissive the statute will not begin to run against the legal owner until an adverse holding is declared and notice of such change is brought to the knowledge of the owner on the other hand it is also well established that the statute of limitations operates against railroad corporations whose lands are held adversely as well as against individuals and this applies to the right of way st louis san francisco railroad company ruttan ark for the error indicated the judgment is reversed and the cause remanded for new trial»

SECOND MOST (47187, 0.36741557717323303): «lappellants larry and julius edge edgeâ or edgesâ appeal the order of the cleburne county circuit court that rejected them request for an easement across the lands of appellees garry bryant and brenda sutherland as co trustees of the sutherland revocable trust dated july sutherland trustâ or bryant sutherlandâ kurt and virginia phillips phillipsâ and garry lynn and angela sutherland lynn sutherlandâ the edges acquired an existing dedicated roadway easement to access their property from the east but they wanted to use different shorter route across these neighboring properties from the south filing suit in april the complaint alleged four counts count sought quiet title as to the correct location of the edge boundary lines count ii and count iii sought an easement by implication or alternatively by necessity over all appelleesâ properties and count iv sought prescriptive easement or ownership by adverse possession of particular portion of land gapâ between the southern boundary of the edge property and the three acres of phillips property after hearing evidence presented by appellants appellees moved for directed verdict appellants do not contest the dismissal of their complaint on counts and iv appellants argue that the trial court erred in granting directed verdict and dismissing their complaint because an easement existed for the gravel drive across the sutherland trust and lynn sutherland properties and because they presented prima facie case of easement by implication or necessity across all the intermediate lands we hold that the trial court did not err in dismissing them complaint seeking an easement to cross all these properties appellants have not demonstrated trial court error we therefore affirm the standard of review is well settled this case was disposed by directed verdict at the close of the plaintiffsâ case in determining on appeal whether directed verdict or dismissal at bench trial was properly entered at the trial court level we review the evidence in the light most favorable to the party against whom the verdict was sought and give it its highest probative value taking into account all reasonable inferences deducible from it nicholson simmons first natâ corp ark motion for sdirected verdict or dismissal should be granted only if there is no substantial evidence to support jury verdict deck house inc link ark app in making that determination the trial court does not engage in fact finding or determine questions of credibility where the evidence is such that fair minded persons might reach different conclusions then fact question is presented and dismissal or directed verdict should be reversed id appellants sought the judicial creation of an easement over land owned by three different persons or entities with the segments of the requested easements created by different judicial mechanisms over two segments appellants requested an easement by implication or by necessity over another segment an express easement by reservation in prior deed or by implication or necessity and over another segment by easement by prescription or implication or necessity with each requested type of easement the law requires different and distinct elements of proof an easement by implication arises where during unity of title landowner imposes an apparently permanent and obvious servitude on part of his property in favor of another part and where at the time of later severance of ownership the servitude is in use and is reasonably necessary for the enjoyment of that part of the property favored by the servitude berry moon ark app in order for such an easement to be established it must appear not only that the easement is obvious and apparently permanent but also that it is reasonably necessary for the enjoyment of the property id the term necessaryâ in this context means that there could be no other reasonable mode of enjoying the dominant tenement without the easement id the necessity for the easement must have existed at the time of the severance id further the apparently permanent nature of the easement must be in existence at the time of common ownership id an easement by necessity arises when there could be no other reasonable mode of enjoying the dominant tenement without the easement horton taylor ark app the possibility of another although inconvenient route to appellantâ property precludes the establishment of an easement by necessity id the degree of necessity thus must be more than mere convenience id to establish an easement by necessity party must prove that at one time one person held title to the tracts in question that unity of title was severed by conveyance of one of the tracts and that the easement is necessary in order for the owner of the dominant tenement to use his land with the necessity existing both at the time of the severance of title and at the time the easement is exercised id an easement by necessity terminates with the cessation of the necessity that brought it into being sluyter hale fireworks pâ ship ark the elements of easement by necessity or implication are very similar prescriptive easement may be gained by one not in fee possession of the land by operation of law in manner similar to adverse possession owners assâ of foxcroft woods inc foxglen assocs ark like adverse possession prescriptive easements are not favored in the law since they necessarily work corresponding losses or forfeitures in the rights of other persons id in arkansas it is generally required that one asserting an easement by prescription show by preponderance of the evidence that oneâ ruse has been adverse to the true owner and under claim of right for the statutory period id our supreme court has said that the statutory period of seven years for adverse possession applies to prescriptive easements id overt activity on the part of the user is necessary to make it clear to the owner of the property that an adverse use and claim are being exerted id mere permissive use of an easement cannot ripen into an adverse claim without clear action which places the owner on notice id the plaintiff bears the burden to show by preponderance of the evidence that there has been adverse not permissive use of the land in question id see also manitowoc remfg inc vocque ark detailed examination of the evidence presented at this bench trial is necessary to decide the issues on appeal julius edge and his son larry edge purchased forty five acres of rural land in march for the purpose of building cabin on it and using the acreage for hunting this land is in the east half of the northeast quarter of section township north range west in cleburne county situated north of wolf bayou road paved roadway although appellants had been granted another route from the eastern side of their property to access wolf bayou road they wanted to access their property from the south by crossing over property owned in varying tracts by appel lees appellee bryant sutherland has owned thirty eight acres directly south of edge for many years wolf bayou road runs adjacent to the southern boundary of bryant sutherlandâ tract squeezed in between the edgesâ acreage and bryant sutherlandâ acreage is three acre rectangular tract owned by appellees kurt and virginia phillips access to the phillips acreage is gained from gravel drive that commences off wolf bayou road and runs generally finortheast to the phillipsâ three acre tract this access is referred to herein as the gravel drive this gravel drive was built by bryant sutherlandâ predecessor in title decades ago the gravel drive however crosses two different tracts of property as the gravel drive begins off wolf bayou road the gravel drive is contained within the west half of the quarter not the east half of the quarter this segment that lies within the west half is referred to herein as the southern portionâ of the gravel drive there is some evidence in the record that this southern portion of the gravel drive is on property owned by lynn sutherland as this gravel drive continues northward it crosses over to the east half of the quarter and lies within the tract owned by bryant sutherland this segment that lie within the east half is referred to herein as the northern portionâ of the gravel drive appellees collectively were unwilling to permit the edges to cross their lands to connect to wolf bayou road as result the edges acquired the aforementioned access from the east over thirty foot wide existing old road easement that had been in existence for many years running more than mile to another point of connection to wolf bayou road it was not ideal for the edges as it was rough hilly terrain that required twice crossing creek their attorney contended that this was not reasonable access to the property this was what led to the lawsuit seeking an easement to cross the neighboring properties from the south the relief requested by appellants was combination of at least four and perhaps five different easements linked together from wolf bayou road to gain access to the edge property from the south as follows these combined easements would complete the desired access route from wolf bayou road to the edge property from the south the trial court heard extensive testimony about the usage of the gravel drive and paths from the phillips acreage toward the property that is now owned by the edges it also heard testimony from edge about the alternative access from the eastern side of the edge acreage whether it was sufficiently passable and why appellants wanted the shorter easier route the attorneys for bryant sutherland lynn sutherland and phillips moved for directed verdict on all counts the trial court found that appellants failed to present sufficient evidence to sustain the burden of proof for easements by necessity or implication on one or more elements and directed verdict in favor of appellees sin reviewing the record on appeal even if we were to hold that there were sufficient facts and inferences to survive motion to dismiss over the segments of the requested easements other than the gravel drive which we do not decide there is an absence of evidence in the record concerning the unity of title regarding the gravel drive without an easement to use the gravel drive the edges cannot reach wolf bayou road our ultimate analysis centers on that segment nowhere in appellantsâ complaint is there an allegation that prescriptive easement was sought or that an express or dedicated easement already existed across the gravel drive the only allegation regarding the gravel drive in the complaint was that an easement by implication or by necessity we affirm»

MEDIAN (40506, 0.08816313743591309): «pro se appellant charles alan baker appeals from the circuit courtâ order dismissing his petition for habeas corpus relief brought against appellee larry norris the director of the arkansas department of correction his sole point on appeal is that the circuit court erred in dismissing his petition we affirm the circuit courtâ order review of the record reveals that bakerâ plea hearing was held in the monroe county circuit court on august at that hearing the circuit court read aloud the following charges against baker possession of controlled substance with intent class possession of controlled substance class possession of firearm by certain person class escape in the second degree class and attempted capital murder class the prosecutor then presented its motion to the circuit court in accord with plea negotiations following the circuit courtâ inquiry as to whether there were any objections counsel for baker stated that there were none and that hat is our understanding the following colloquy then took place the circuit court then addressed baker the circuit court then inquired of baker as to whether he had discussed the potential penalties and his right to jury trial and finally the circuit court inquired of baker as to the facts and whether he did in fact commit the offense and also sentenced him in accordance with his negotiated plea subsequent to the hearing judgment and commitment order was entered on august reflecting that baker entered negotiated plea of guilty to the offense of possession of drug paraphernalia with intent to manufacture methamphetamine in violation of ark code ann pamph no sept and that he was sentenced to monthsâ imprisonment on august the department of correction completed time computation card that only reflected bakerâ prior conviction for criminal conspiracy it indicated that he was eligible for transfer on january and had discharge date of december on october an amended judgment and commitment order was entered by the circuit court reflecting that baker had negotiated plea of guilty to the offense of possession of drug paraphernalia with intent to manufacture methamphetamine in violation of ark code ann pamph no sept and sentencing him to monthsâ imprisonment emphasis added on november the department of correction completed another time computation card which reflected the addition of his drug paraphernalia conviction accordingly bakerâ new eligibility date for transfer was august and his discharge date was computed to be february on march baker filed an affidavit in support of his pro se petition for writ of habeas corpus in the jefferson county circuit court in it he claimed that on august he agreed to enter guilty plea to the charge of criminal conspiracy to manufacture and delivery and possession of methamphetamine he stated that he was told that he would be sentenced and that he would plead guilty to the charge under ark code ann and that he was advised that he would receive fifteen year sentence to be served concurrently with prior conviction he claimed that he was told that he would be required to serve only one half of his sentence minus statutory credits for good time and that he would receive days of credit for time served he specifically stated that he was never informed by his trial counsel the prosecuting attorney the trial court before or at the guilty plea trial that he would be pleading guilty to violation that ould require him to serve of his sentence before being parole eligible finally he asserted that neither he nor his counsel was present when the trial court changed the nature of the sentence origionally sic imposed and amended the judgment so that his conviction would be pursuant to rather than under the statute that he agreed to plead guilty to nor did they receive notice that the order would be changed on march baker filed his petition for issuance of writ of habeas corpus in it he claimed that the monroe county circuit court lacked jurisdiction to amend his judgment and commitment order without notice to him and without him being afforded the right to be present and represented by counsel he further asserted that the amended judgment and commitment order clearly prejudiced him by altering the nature and degree of his punishment in addition he claimed that because he had already pled guilty to violation of ark code ann the amended judgment and commitment order convicting him under ark code ann violated his double jeopardy rights for these reasons he claimed the amended judgment and commitment order was invalid on its face on april the attorney generalâ office on behalf of appellee norris filed motion to dismiss bakerâ petition in it the state conceded that the original judgment and commitment order did list the incorrect code section under which baker was sentenced however the state asserted the statutory reference was clerical error which was corrected by the amended judgment and commitment order to properly reflect the offense to which baker actually pled guilty based on the clerical nature of the correction the state averred that bakerâ petition for habeas relief should be dismissed with prejudice as he did not challenge the facial invalidity of the amended judgment and commitment order and because the jurisdiction of the circuit court was not at issue baker replied asserting that the original judgment and commitment order controlled and that the circuit court lacked jurisdiction to enter the amended judgment and commitment order on june the jefferson county circuit court filed its order dismissing bakerâ petition in it the circuit court concluded that bakerâ allegations did not demonstrate that the monroe county circuit court lacked jurisdiction or that the commitment was invalid on its face the circuit court found that the monroe county circuit court had personal jurisdiction over baker and jurisdiction of the subject matter and thus it had the authority to render the judgment moreover the circuit court found that baker had not stated any facts or advanced any evidence to support his claim for these reasons the circuit court concluded that bakerâ petition was frivolous and without merit and dismissed the petition with prejudice in addition the circuit court found that the dismissal should be considered strikeâ within the meaning of ark code ann repl baker now brings this appeal baker argues that in the instant case he pled guilty to violation of ark code ann and not ark code ann he claims that his fifteen year sentence was valid sentence under section and that once it was put into execution the circuit court lost jurisdiction to amend the judgment and commitment order especially without notice to him and his counsel he asserts that the amendment prejudiced him by requiring him to serve seventy percent of his sentence as opposed to the quarter of his sentence which he believed would be the case after his plea in addition he claims that the amended judgment and commitment order violated his right against double jeopardy the state on behalf of appellee norris responds that baker failed to meet his burden of showing any basis for finding that writ of habeas corpus should issue while the state concedes that the original judgment and commitment order does list the incorrect criminal code section under which baker was sentenced it urges that the sentencing court not only had jurisdiction to commit baker to imprisonment but also had personal jurisdiction over baker and had jurisdiction over the subject matter as well as authority to accept the guilty plea and to render judgment it submits that the amended judgment and commitment order was facially valid and that because circuit court may correct clerical error the sentencing court did not lack jurisdiction to amend the original judgment and commitment order writ of habeas corpus is proper when judgment of conviction is invalid on its face or when circuit court lacks jurisdiction over the cause see noble norris ark unless petitioner can show that the circuit court lacked jurisdiction or that the commitment was invalid on its face there is no basis for finding that writ of habeas corpus should issue see id the petitioner must plead either the facial invalidity or the lack of jurisdiction and make showing by affidavit or other evidence of probable cause to believeâ he or she is illegally detained id quoting ark code ann repl moreover habeas proceeding does not afford prisoner an opportunity to retry his or her case and it is not substitute for direct appeal or postconviction relief see id hearing is not required if the petition does not allege either of the bases of relief proper in habeas proceeding and even if cognizable claim is made the writ does not have to be issued unless probable cause is shown see id lastly an appeal is the proper procedure for the review of circuit courtâ denial of petition for writ of habeas corpus see id in the instant case baker challenges the sentencing courtâ jurisdiction to amend his judgment and commitment order to reflect the correct statute under which he pled guilty jurisdiction is the power of the court to hear and determine the subject matter in controversy see id while baker claims that he pled guilty to violation of ark code ann review of the record plainly reveals that he pled guilty to the offense of possession of drug paraphernalia with the intent to manufacture methamphetamine that offense is violation of ark code ann not section section clearly provides ark code ann pamph no sept accordingly bakerâ original judgment and commitment order was erroneous as it listed his offense as violation of section the question becomes then whether the sentencing court had jurisdiction to amend the judgment and commitment order to reflect the correct statute we hold that it did we have held that our case law is replete with examples of clerical error in judgment and commitment order and that such clerical errors have not prevented enforcement of the judgment and commitment order see carter norris ark as clerical errors do not speak the truth courts have the power to enter an amended judgment and commitment order nunc pro tunc to correct an erroneous judgment see id citing ark civ mccuen state ark willis state ark app here baker admitted to possessing drug paraphernalia with the intent to manufacture methamphetamine during his plea hearing the mere correction of the of offenseâ on the judgment and commitment order was the correction of clerical error especially where the name of the offense its seriousness level and all other information were correct and remained the same the principal issue in habeas corpus proceeding is whether the petitioner is detained without lawful authority see carter norris supra in bakerâ petition to the circuit court he failed to demonstrate that he was detained without lawful authority because he failed to show that his commitment was invalid on its face or failed to show that the sentencing court lacked jurisdiction he did not establish any cause to grant the writ of habeas corpus for that reason we affirm the circuit courtâ denial of habeas relief baker additionally argues that the habeas court erred in designating the dismissal of his petition with prejudice as strikeâ under ark code ann repl ark code ann the state asserts that this court should not address this argument because baker has failed to cite any authority in support of his argument we agree we have held that we will not consider an argument on appeal that has no citation to authority or convincing legal argument see johnson state ark indeed we will not research or develop an argument for an appellant see flanagan state ark accordingly we decline to address his argument affirmed»

LEAST (29578, -0.1317369043827057): «»